IdentitySdkCore/Podfile

```diff
@@ -3,12 +3,20 @@ source 'https://cdn.cocoapods.org/'
 # Comment the next line to not define a global platform for your project
 platform :ios, '12'
 
+inhibit_all_warnings!
```

Comment:
Could you explain which warnings we're inhibiting? I don't know much about Swift/iOS development, but will this inhibit warnings that might be useful to SDK integrators?

Reply:
yes, I forgot to delete this line, I added it in the circle CI conf, it was to avoid the warning of the team certification.